### ISSUE TITLE
P2201 Mixed string literal concatenation

### ISSUE THREAD

**Issue by wg21bot** [cplusplus/papers#905]
[P2201R0](https://wg21.link/p2201r0) Mixed string literal concatenation (Jens Maurer)

**Comment by tahonermann** [comment]
SG16 reviewed [P2201R0](https://wg21.link/p2201r0) during a [telecon held July 22nd, 2020](https://github.com/sg16-unicode/sg16-meetings#july-22nd-2020) and had unanimous support to forward the paper.  The poll taken was:

**Poll: Direct Tom to recommend to the EWG chair that P2201R0 be forwarded directly to the CWG.**

| SF | F | N | A | SA |
| -- | -- | -- | -- | -- |
| 8 | 1 | 0 | 0 | 0 |

The poll to forward directly to the CWG was motivated by the fact that the proposal seeks only to 1) standardize what is already existing practice among all the major C++ implementors, and 2) change an existing feature from conditionally-supported implementation-defined behavior to ill-formed.  The intent was to avoid spending time in the EWG for a change that is arguably not evolutionary in nature.

It is now up to the EWG chair to accept the SG16 recommendation and forward to the CWG, or to allocate time in the EWG to discuss and affirm the proposal.  Removing the SG16 label and adding EWG.

**Comment by jfbastien** [comment]
Discussed in the EWG telecon today:

**POLL*: P2201 can go directly to CWG, after adding an Annex C entry.

SF| F| N| A| SA
-|-|-|-|-
10| 7| 4| 0| 0

**Comment by AaronBallman** [comment]
This paper does not need to be seen by SG22 as it was already seen by WG14 as part of [WG14 N2594](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n2594.htm), which was approved at the Dec 2020 meeting.

Straw Poll: Does the committee wish to adopt N 2594 into C23 as is? 18-0-2 passes

**Comment by jfbastien** [comment]
[EWG February 2021 polls](https://isocpp.org/files/papers/P1018R9.html#P2201R0):

Poll: Forward P2201R0 "Mixed string literal concatenation" to Core, after adding an Annex C entry.

SF | F | N | A | SA
-- | -- | -- | -- | --
14 | 12 | 1 | 0 | 0

Poll outcome: ✅ consensus.

Salient comments:

* This was meaningless and unimplemented. No good reason to have it.
* I was very horrified that we allowed this when I saw this paper for the first time.
* As paper said, this is not portable and without clear meaning.
* Implementation defined behavior that doesn’t have many supporting implementations, PLUS can only result in nonsense needs to disappear from the language.
